.travis.yml

```diff
@@ -2,18 +2,22 @@ dist: xenial
 sudo: required
 language: java
 jobs:
+  fast_finish: true
   include:
   - os: linux
-    dist: bionic
+    jdk: openjdk8
+  - os: linux
+    jdk: openjdk11
+  - bridge: s390xbuild
```

Comment:
Out of curiosity ... what does this `bridge: s390xbuild` mean?

Reply:
You need an identifier for the allow failures section to specify which job is allowed to fail.

Reply:
I wonder if this information would be useful as a comment?

Always nice when an identifier string is documented, avoids people googling and being confused in the future.

Reply:
Yes good idea, I added a comment.